internal/pkg/aws/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudwatchlogs contains utility functions for Cloudwatch Logs client.
+package cloudwatchlogs
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+)
+
+const (
+	// SleepDuration is the sleep time for making the next request for log events.
+	SleepDuration        = 1 * time.Second
+	orderByLastEventTime = "LastEventTime"
```

Comment:
You can use the SDK const for this `cloudwatchlogs.OrderByLastEventTime`

Reply:
Oh yes! I missed the previous comment. Sorry about that...